src/app/pages/wallets/wallet-details/pages-wallet-details.component.html

```diff
@@ -0,0 +1,61 @@
+<div class="container walletDetailsContainer">
+
+  <div class="wallet__description">
+    <div class="name">
+      <div class="name__name">
+        {{ walletsManagementItem?.name }}
+      </div>
+      <div class="name__edit">
+        <button mat-stroked-button color="primary" (click)="handleWalletEdit()">Edit</button>
+      </div>
+    </div>
+
+    <div class="newTransaction">
+      <button mat-flat-button color="primary">Add new transaction</button>
+    </div>
+  </div>
+
+  <div class="wallet__sort">
+    <mat-form-field appearance="fill">
+      <mat-label>Choose Transactions</mat-label>
```

Comment:
Lets change it to "Select type" -> also it should have `All transactions` selected by default.